engine/src/flutter/display_list/dl_storage.cc

```diff
@@ -10,6 +10,22 @@ static constexpr inline bool is_power_of_two(int value) {
   return (value & (value - 1)) == 0;
 }
 
+static uint32_t NextPowerOfTwoSize(uint32_t x) {
```

Comment:
Needs to be size_t and deal with up to 64-bits.

Reply:
Is size_t 64-bits on all platforms? If not then it needs to declare size_t and then have an optional extra set of shifts and ors depending on the sizeof size_t.